libs/coin-framework/src/bridge/jsHelpers.test.ts

```diff
@@ -0,0 +1,40 @@
+import BigNumber from "bignumber.js";
+import { defaultUpdateTransaction } from "./jsHelpers";
+import type { TransactionCommon } from "@ledgerhq/types-live";
+
+describe("jsHelpers", () => {
+  describe("defaultUpdateTransaction", () => {
+    it("should not update the transaction object", () => {
+      const transaction: TransactionCommon = {
+        recipient: "0x17733CAb76d9A2112576443F21735789733B1ca3",
+        amount: new BigNumber("10000000000000"),
+      };
+
+      const updatedTransaction = defaultUpdateTransaction(transaction, {
+        recipient: "0x17733CAb76d9A2112576443F21735789733B1ca3",
+        amount: new BigNumber("10000000000000"),
+      });
+
+      expect(Object.is(transaction, updatedTransaction)).toBe(true);
```

Comment:
You can use `.toBe` instead of `Object.is` because `.toBe` is already using `Object.is` under the hood.

Reply:
Done here c1cfc6d076467b2b15b699e50b3fecb29cba1dbd
Got confused by the last paragraph of the [Jest `toBe` doc](https://jestjs.io/docs/expect#tobevalue) that is actually only related to debug